doc/classes/DisplayServer.xml

```diff
@@ -1908,7 +1908,7 @@
 			Display server supports windows can use per-pixel transparency to make windows behind them partially or fully visible. [b]Windows, macOS, Linux (X11/Wayland)[/b]
 		</constant>
 		<constant name="FEATURE_HIDPI" value="12" enum="Feature">
-			Display server supports querying the operating system's display scale factor. This allows for [i]reliable[/i] automatic hiDPI display detection, as opposed to guessing based on the screen resolution and reported display DPI (which can be unreliable due to broken monitor EDID). [b]Windows, Linux (Wayland), macOS[/b]
+			Display server supports querying the operating system's display scale factor. This allows to automatically detect the hiDPI display [i]reliably[/i], instead of guessing based on the screen resolution and the display's reported DPI (which may be unreliable due to broken monitor EDID). [b]Windows, Linux (Wayland), macOS[/b]
```

Comment:
```suggestion
			Display server supports querying the operating system's display scale factor. This allows automatically detecting the hiDPI display [i]reliably[/i], instead of guessing based on the screen resolution and the display's reported DPI (which may be unreliable due to broken monitor EDID). [b]Windows, Linux (Wayland), macOS[/b]
```